src/Core/Configuration.php

```diff
@@ -44,6 +44,7 @@ class Configuration
     ];
 
     private const DEFAULT_OPTIONS = [
+        self::OPTION_REGION => 'us-east-1',
```

Comment:
Some services are global and don't need region (ie. STS).

I hesitate with:
- fallback to `us-east-1` when the Service is global
- trigger an exception when the service requires a region.

WDYT?

Reply:
This is fine =)